interface/forms/newpatient/templates/newpatient/partials/common/_head.html.twig

```diff
@@ -0,0 +1,18 @@
+<head>
+    {% include "newpatient/partials/common/_head-before.html.twig" %}
+    <title>{{ pageTitle|text }}</title>
+    {# datetimepicker used for encounter date pickers #}
+    {# moment(moment.js) & validation_script used for script validation, with moment needed for date validators #}
+    {{ setupHeader(['datetime-picker', 'datetime-picker-translated', 'common', 'moment', 'validation_script'])|attr }}
```

Comment:
note sure why |attr is above

Reply:
Shouldn't have been there, not sure how it got in lol.  Good catch.

Reply:
Fixed.